domains/transfer/Transfer.API/API/Cvr/Api/v2024_01_03/Controllers/CvrController.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using API.Cvr.Api.Clients.Cvr;
+using API.Cvr.Api.Models;
+using API.Cvr.Api.v2024_01_03.Dto.Responses;
+using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Cvr.Api.v2024_01_03.Controllers;
+
+[Authorize]
+[ApiController]
+[ApiVersion(20240103)]
+[Route("api/cvr")]
+public class CvrController(CvrClient client) : Controller
+{
+    /// <summary>
+    /// Get CVR registered company information for multiple CVR numbers
+    /// </summary>
+    /// <response code="200">Successful operation</response>
+    /// <response code="404">CVR company not found</response>
+    [ProducesResponseType(typeof(CvrCompanyListResponse), 200)]
+    [ProducesResponseType(typeof(void), 404)]
```

Comment:
It returns 200 OK with an empty collection if cvr company not found

Reply:
Changed

Reply:
https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/893686fdbe2f715060dd6372f2b06652f4e0243f